android/engine/src/main/java/org/smartregister/fhircore/engine/rulesengine/RulesFactory.kt

```diff
@@ -288,6 +289,9 @@ constructor(
     /** This function extracts the patient's age from the patient resource */
     fun extractAge(patient: Patient): String = patient.extractAge(context)
 
+    /** This function extracts the patient's phone from the patient resource */
+    fun extractPhone(patient: Patient): String = patient.extractPhone(context)
```

Comment:
I do not think this is really necessary. This extraction can be done via fhirPath.

Reply:
got it, 
fixed with 
**fhirPath.extractValue(Patient, 'Patient.telecom[0].value')**
pushing in config